brands/leisure/fitness_centre.json

```diff
@@ -76,6 +76,16 @@
       "name": "Fitness First"
     }
   },
+  "leisure/fitness_centre|Fitness24Seven": {
+    "countryCodes": ["fi", "sv"],
```

Comment:
It is supposed to be Finland, Sweden, right? Note that `sv` is a language code for Swedish language, and country code for El Salvador.

Sweden has `se` country code.

Reply:
That is supposed to be Sweden, yes. I'll fix that.

Reply:
Thanks for fixing!